packages/test/service-load-test/src/nodeStressTest.ts

```diff
@@ -79,132 +76,189 @@ function createLoader(config: IConfig, password: string) {
     return loader;
 }
 
-async function initialize(config: IConfig, driveId: string, password: string) {
-    const loader = createLoader(config, password);
+async function initialize(driveId: string, loginInfo: IOdspTestLoginInfo) {
+    const loader = createLoader(loginInfo);
     const container = await loader.createDetachedContainer(codeDetails);
     container.on("error", (error) => {
         console.log(error);
         process.exit(-1);
     });
-    const tenant = `https://${config.server}`;
-    const request = urlResolver.createCreateNewRequest(tenant, driveId, "/test", "test");
+    const siteUrl = `https://${loginInfo.server}`;
+    const request = urlResolver.createCreateNewRequest(siteUrl, driveId, "/test", "test");
     await container.attach(request);
     const dataStoreUrl = await container.getAbsoluteUrl("/");
     console.log(dataStoreUrl);
+    assert(dataStoreUrl);
+
     container.close();
 
     return dataStoreUrl;
 }
 
-async function load(config: IConfig, url: string, password: string) {
-    const loader = createLoader(config, password);
+async function load(loginInfo: IOdspTestLoginInfo, url: string) {
+    const loader = createLoader(loginInfo);
     const respond = await loader.request({ url });
     // TODO: Error checking
     return respond.value as ILoadTest;
 }
 
-async function getDriveId(server: string, odspTokens: IOdspTokens): Promise<string> {
+/**
+ * Query the SharePoint API for the default drive (based on server the user claims in the tokens) and return its ID
+ */
+async function getDefaultDriveId(server: string, odspTokens: IOdspTokens): Promise<string> {
     const driveResponse = await getAsync(
         `https://${server}/_api/v2.1/drive`,
         { accessToken: odspTokens.accessToken },
     );
 
     const driveJson = await driveResponse.json();
-
     return driveJson.id as string;
 }
 
 async function main() {
-    let config: IConfig;
-    try {
-        config = JSON.parse(fs.readFileSync("./testConfig.json", "utf-8"));
-    } catch (e) {
-        console.error("Failed to read testConfig.json");
-        console.error(e);
-        process.exit(-1);
-    }
-
     commander
         .version("0.0.1")
-        .requiredOption("-w, --password <password>", "Password for username provided in testconfig.json")
+        .requiredOption("-t, --tenant <tenant>", "Which test tenant info to use from testConfig.json", "fluidCI")
         .requiredOption("-p, --profile <profile>", "Which test profile to use from testConfig.json", "full")
+        // eslint-disable-next-line max-len
+        .option("-w, --password <password>", "Password for username provided in testconfig.json. Falls back to checking login__odsp__test__accounts from env")
```

Comment:
i'm not even sure we want to support this. it's generally not secure to take passwords via cmd line as it's hard to prevent them from being logged

Reply:
That's a good point.  I'll do that in a follow-up, I will want to highlight that change on its own for our partners' sake who are starting to use this tool more.